docs/developer-guide/syntaxes.md

```diff
@@ -1,33 +1,42 @@
 # Writing custom syntaxes
 
-Custom syntaxes are [PostCSS syntaxes](https://github.com/postcss/postcss#syntaxes) written by the community to support other styling languages or CSS-in-JS libraries using the [`customSyntax` option](../user-guide/usage/options.md#customSyntax)
+Custom syntaxes are [PostCSS syntaxes](https://github.com/postcss/postcss#syntaxes) written by the community to support other styling languages and CSS-in-JS libraries using the [`customSyntax` option](../user-guide/usage/options.md#customSyntax)
 
 To write one, familiarize yourself with PostCSS's [how to write custom syntax](https://github.com/postcss/postcss/blob/main/docs/syntax.md) guide.
 
 Existing syntaxes that you can use for reference include:
 
 - [postcss-scss](https://github.com/postcss/postcss-scss)
 - [postcss-less](https://github.com/shellscape/postcss-less)
```

Comment:
[suggest] How about keeping consistency with the links (npmjs.com or github.com)?

```suggestion
- [postcss-scss](https://www.npmjs.com/package/postcss-scss)
- [postcss-less](https://www.npmjs.com/package/postcss-less)
```